packages/sdk/src/adapters/standard-bridge.ts

```diff
@@ -13,8 +13,10 @@ import {
   BlockTag,
 } from '@ethersproject/abstract-provider'
 import { predeploys } from '@eth-optimism/contracts'
-import { getContractInterface } from '@eth-optimism/contracts-bedrock'
 import { hexStringEquals } from '@eth-optimism/core-utils'
+import l1StandardBridgeArtifact from '@eth-optimism/contracts-bedrock/forge-artifacts/L1StandardBridge.sol/L1StandardBridge.json'
+import l2StandardBridgeArtifact from '@eth-optimism/contracts-bedrock/forge-artifacts/L2StandardBridge.sol/L2StandardBridge.json'
+import optimismMintableERC20 from '@eth-optimism/contracts-bedrock/forge-artifacts/OptimismMintableERC20.sol/OptimismMintableERC20.json'
```

Comment:
We could clean up these imports if we wanted with [exports](https://github.com/wagmi-dev/wagmi/blob/main/packages/react/package.json#L30) in a future pr

Reply:
Would love to clean this up in a future pr